### PR TITLE
Require emulated TPM2 for Stratis tests

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

**Comment by mulkieran** [comment]
All our tests failed w/ Artemis, guest couldn't be provisioned error. Cockpit tests failed with some error that is surely unrelated to stratisd changes. So we can be pretty sure that it is the configuration changes, for our tests only, that are breaking something.

**Comment by mulkieran** [comment]
Ok. So this approach is guaranteed not to work. We would probably have to provision in the normal way and then use swtpm to emulate the TPM2 module on the provided virtual machine.

**Comment by mulkieran** [comment]
our tests seem to be stuck running with no information at all.

**Comment by mulkieran** [comment]
I failed to background a step...

**Comment by mulkieran** [comment]
shell syntax  interpretation problems; I think this attempt may work

**Comment by mulkieran** [comment]
rebased

File: plans/all.fmf [hunk 1]
```diff
@@ -19,6 +28,9 @@ prepare:
   - name: Show test system information
     how: shell
     script: free -m; lsblk -i; lscpu; cat /proc/1/sched
+  - name: Show TPM2 information
+    how: shell
+    script: tpm2_getrandom 16 | hexdump
```
Comment (jbaublitz) [inline plans/all.fmf:33]: Should this go in the final version? I thought it was just for testing that the TPM was correctly provisioned during development of the PR.

**Review by jbaublitz — CHANGES_REQUESTED** [review]

File: plans/all.fmf (hunk 1 above)
Comment (mulkieran) [reply plans/all.fmf:33]: I don't mind dropping it.

**Comment by mulkieran** [comment]
@jbaublitz This is ready again.

**Review by jbaublitz — APPROVED** [review]

**Pull request merged**